packages/checkout/widgets-lib/src/resources/text/textConfig.ts

```diff
@@ -458,6 +458,11 @@ export const text = {
       layoutHeading: 'In progress',
       passportDashboard: 'View the full transaction history in your',
       status: {
+        withdrawalPending: {
```

Comment:
These need to be added into the widgets-lib locales en.JSON and other language files as well.

Reply:
updated this now